docs/content/reference.md

```diff
@@ -151,6 +151,28 @@ Bolt's client is an instance of `WebClient` from the [Node Slack SDK](https://to
 
 :::
 
+## Assistants
```

Comment:
I know this doesn't matter the rest of the doc structure but idk what else to call all this

Reply:
For consistency, we may want to use Agents & Assistants here too?